app/assets/stylesheets/vendor/autoSuggest.css

```diff
@@ -2,18 +2,16 @@
 
 ul.as-selections {
 	list-style-type: none;
-	border: 1px solid #ccc;
 	margin: 0;
 	overflow: auto;
 	background-color: #fff;
 	border-radius: 3px;
-	/* 1% padding (all sides) + 98% width = 100% */
-	padding: 1%;
-	width: 98%;
+	width: 100%;
+  padding: 0;
```

Comment:
On second thought it seems to be upstream code, so ideally you would not add (further) changes but provide overrides instead.